akka-actor/src/main/scala/akka/dispatch/Dispatcher.scala

```diff
@@ -41,6 +44,10 @@ class Dispatcher(
     def copy(): LazyExecutorServiceDelegate = new LazyExecutorServiceDelegate(factory)
   }
 
+  /**
+   * Updated via esUpdater
```

Comment:
?

Reply:
At first glance this `var` would not seem to be updated anywhere - but in fact it is, via the esUpdater `AtomicReferenceFieldUpdater` below - I'll make the comment a bit clearer :)

Reply:
Ah, I thought it was a comment added by a tool of some kind